R/transformCounts.R

```diff
@@ -40,10 +40,26 @@
 #' \item{'clr'}{ Centered log ratio (clr) transformation can be used for reducing the
 #' skewness of data and for centering it. (See e.g. Gloor et al. 2017.)
 #'
-#' \deqn{clr = log_{10}x_{r} - log_{10}µ_{r}}{%
-#' clr = log10 x_r - log10 µ_r}
-#' where \eqn{x_{r}} is a single relative value, \eqn{\mu_{r}} is
-#' mean relative value".}
+#' \deqn{clr = log_{10}\frac{x_{r}}{g(x_{r})} = log_{10}x_{r} - log_{10}µ_{r}}{%
+#' clr = log10(x_r/g(x_r)) = log10 x_r - log10 µ_r}
+#' where \eqn{x_{r}} is a single relative value, g(x_{r}) is geometric mean of
+#' sample-wide relative values, and \eqn{\mu_{r}} is arithmetic mean of 
+#' sample-wide relative values".}
+#' 
+#' \item{'clr_robust'}{ Robust clr is similar to regular clr. Problem of regular
```

Comment:
I suggest to use the established abbreviation rclr (for "robust clr"). Shorter to write and links more explicitly with some other implementations.

Reply:
ok